pkg/migrator/migrator.go

```diff
@@ -40,39 +40,39 @@ func New(client client.Client) Migrator {
 }
 func (m *migrator) Migrate() error {
 
-	cscs := &v2.CatalogSourceConfigList{}
-	err := m.client.List(context.TODO(), &client.ListOptions{}, cscs)
+	opsrcs := &v1.OperatorSourceList{}
+	err := m.client.List(context.TODO(), &client.ListOptions{}, opsrcs)
 	if err != nil {
 		return err
 	}
 	allErrors := []error{}
-	for _, csc := range cscs.Items {
-		err = removeCSCOwnerRefFromCatalogSource(&csc, m.client)
+	for _, opsrc := range opsrcs.Items {
+		err = removeOpsrcOwnerRefFromCatalogSource(&opsrc, m.client)
 		if err != nil {
 			allErrors = append(allErrors, err)
```

Comment:
Might be worth checking if the OpSrc API exists before running this code so we don't populate 404 errors.